include/ximea_driver/ximea_driver.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <memory>
+
+#include "m3api/xiApi.h"
+
+#include "sensor_msgs/msg/image.hpp"
+#include "rclcpp/rclcpp.hpp"
+
+namespace trailbot {
+
+class XIMEADriver : public rclcpp::Node {
+public:
+  XIMEADriver();
+
+  std::unique_ptr<sensor_msgs::msg::Image> GetImage();
+
+private:
+  HANDLE xi_handle_;
```

Comment:
should this variable be private if it is being accessed in a cpp file?

Reply:
It's private so that it's local to the class. Class methods defined in other files can still access it.